lte/gateway/python/setup.py

```diff
@@ -102,6 +102,7 @@
         'eventlet>=0.24',
         'h2>=3.2.0',
         'hpack>=3.0',
+        'freezegun'
```

Comment:
for this lets try to declare it on a specific version of the library, we've seen some issues when there's implicit dependencies on other libraries while installing the magma pkg